packages/suite-web/src/index.ts

```diff
@@ -1,22 +1,25 @@
+import './polyfills';
```

Comment:
I'll try to include these polyfills through vite rather than in the source code / will make a simple vite entrypoint for it so that the source code isn't poluted